support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -381,15 +372,14 @@ export function ThreeTierLanding({
 	});
 	const tier1Link = `checkout?${tier1UrlParams.toString()}`;
 
-	const tier1Card = {
-		productDescription: productCatalogDescription.Contribution,
+	const tier1Card: CardContent = {
+		product: 'Contribution',
 		price: recurringAmount,
 		link: tier1Link,
 		isUserSelected:
 			urlSearchParamsProduct === 'Contribution' ||
 			isCardUserSelected(recurringAmount),
-		isRecommended: false,
-		ctaCopy: 'Support',
+		...settings.products.Contribution,
```

Comment:
Like the way you append this onto `CardContent` to be used later (like `title` further down)